dotcom-rendering/src/components/SupportingContent.tsx

```diff
@@ -59,8 +59,11 @@ const liStyles = css`
 	flex: 1;
 	padding-top: 2px;
 	position: relative;
-	margin-top: 8px;
+	&:first-child {
+		margin-top: 8px;
+	}
 	${from.tablet} {
+		margin-top: 8px;
 		margin-bottom: 4px;
```

Comment:
As mentioned in #8425, `padding` will ensure that the area is clickable, so you might want to consider using it here.

Reply:
I dont think we want that part to be clickable? This is above the <li>